provision/acc_provision/templates/aci-containers.yaml

```diff
@@ -219,7 +219,7 @@ metadata:
 data:
   controller-config: |-
     {
-        "flavor": {{ config.flavor }},
+        "flavor": "{{ config.flavor }}",
```

Comment:
config.flavor|json  can be used as mentioned by @snaiksat , to have a json string

Reply:
Changed